-- MySQL dump 10.13  Distrib 8.0.39, for Linux (x86_64)
--
-- Host: localhost    Database: your_database_name
-- ------------------------------------------------------
-- Server version	8.0.39-0ubuntu0.22.04.1

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Dumping data for table `Ensembl_table`
--

LOCK TABLES `Ensembl_table` WRITE;
/*!40000 ALTER TABLE `Ensembl_table` DISABLE KEYS */;
INSERT INTO `Ensembl_table` VALUES ('ENSMUSG00000000555',16402,'Itga5',15,'protein_coding',103252713,103275190,-1),('ENSMUSG00000001655',15422,'Hoxc13',15,'protein_coding',102829538,102837249,1),('ENSMUSG00000001657',15426,'Hoxc8',15,'protein_coding',102899039,102902543,1),('ENSMUSG00000001661',15425,'Hoxc6',15,'protein_coding',102906689,102920313,1),('ENSMUSG00000022485',15424,'Hoxc5',15,'protein_coding',102875878,102925861,1),('ENSMUSG00000023048',66151,'Prr13',15,'protein_coding',102367463,102371241,1),('ENSMUSG00000023055',67488,'Calcoco1',15,'protein_coding',102615212,102630613,-1),('ENSMUSG00000036061',71726,'Smug1',15,'protein_coding',103061717,103075519,-1),('ENSMUSG00000075394',15423,'Hoxc4',15,'protein_coding',102927366,102945278,1),('ENSMUSG00000076010',751557,'Mir615',15,'miRNA',102923342,102923433,1);
/*!40000 ALTER TABLE `Ensembl_table` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `NCBI_table`
--

LOCK TABLES `NCBI_table` WRITE;
/*!40000 ALTER TABLE `NCBI_table` DISABLE KEYS */;
INSERT INTO `NCBI_table` VALUES (16402,'genomic','15 58.9 cM','integrin alpha 5 (fibronectin receptor alpha)','The product of this gene belongs to the integrin alpha chain family. Integrins are heterodimeric integral membrane proteins composed of an alpha chain and a beta chain. This gene encodes the integrin alpha 5 chain, which is proteolytically processed to generate light and heavy chains that join with beta 1 to form a fibronectin receptor. In addition to adhesion, integrins are known to participate in cell-surface mediated signaling. Integrin alpha 5 and integrin alpha V chains are produced by distinct genes. Homozygous knockout mice for this gene exhibit embryonic lethality. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2015]',31,'Mus musculus'),(15422,'genomic','15','homeobox C13','Enables DNA-binding transcription activator activity, RNA polymerase II-specific and chromatin binding activity. Acts upstream of or within several processes, including hair follicle development; nail development; and tongue morphogenesis. Located in nucleus. Is expressed in several structures, including integumental system; limb; spinal cord; tail; and tongue. Human ortholog(s) of this gene implicated in ectodermal dysplasia 9. Orthologous to human HOXC13 (homeobox C13). [provided by Alliance of Genome Resources, Nov 2024]',2,'Mus musculus'),(15426,'genomic','15 58.05 cM','homeobox C8','Enables DNA binding activity. Acts upstream of or within several processes, including anterior/posterior pattern specification; negative regulation of transcription by RNA polymerase II; and skeletal system morphogenesis. Located in nucleus. Is expressed in several structures, including alimentary system; embryo mesenchyme; genitourinary system; integumental system; and nervous system. Orthologous to human HOXC8 (homeobox C8). [provided by Alliance of Genome Resources, Nov 2024]',3,'Mus musculus'),(15425,'genomic','15 58.06 cM','homeobox C6','Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Acts upstream of or within anterior/posterior pattern specification and embryonic skeletal system development. Predicted to be located in cytosol and nucleoplasm. Predicted to be active in nucleus. Is expressed in several structures, including alimentary system; embryo mesenchyme; genitourinary system; musculoskeletal system; and nervous system. Orthologous to human HOXC6 (homeobox C6). [provided by Alliance of Genome Resources, Nov 2024]',4,'Mus musculus'),(15424,'genomic','15 58.07 cM','homeobox C5','Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Acts upstream of or within anterior/posterior pattern specification and embryonic skeletal system development. Predicted to be located in cell junction and nucleoplasm. Predicted to be active in nucleus. Is expressed in several structures, including alimentary system; central nervous system; embryo mesenchyme; genitourinary system; and respiratory system. Orthologous to human HOXC5 (homeobox C5). [provided by Alliance of Genome Resources, Nov 2024]',2,'Mus musculus'),(66151,'genomic','15 57.6 cM','proline rich 13','Predicted to be located in cytosol. Predicted to be active in nucleoplasm. Orthologous to human PRR13 (proline rich 13). [provided by Alliance of Genome Resources, Nov 2024]',5,'Mus musculus'),(67488,'genomic','15 57.81 cM','calcium binding and coiled coil domain 1','Enables chromatin binding activity and transcription coactivator activity. Involved in nuclear receptor-mediated steroid hormone signaling pathway and positive regulation of transcription by RNA polymerase II. Located in chromatin. Is expressed in several structures, including cerebral cortex; choroid plexus; and thymus primordium. Orthologous to human CALCOCO1 (calcium binding and coiled-coil domain 1). [provided by Alliance of Genome Resources, Nov 2024]',16,'Mus musculus'),(71726,'genomic','15 58.2 cM','single-strand selective monofunctional uracil DNA glycosylase','Predicted to enable DNA N-glycosylase activity; identical protein binding activity; and single-stranded DNA binding activity. Predicted to be involved in base-excision repair. Predicted to be located in cytosol; fibrillar center; and nucleoplasm. Is expressed in several structures, including lower urogenital tract; palatal shelf; reproductive system; retina layer; and urethra epithelium. Orthologous to human SMUG1 (single-strand-selective monofunctional uracil-DNA glycosylase 1). [provided by Alliance of Genome Resources, Nov 2024]',12,'Mus musculus'),(15423,'genomic','15 58.08 cM','homeobox C4','Enables HMG box domain binding activity. Acts upstream of or within anterior/posterior pattern specification; cartilage development; and embryonic organ morphogenesis. Predicted to be active in nucleoplasm. Is expressed in several structures, including alimentary system; embryo mesenchyme; nervous system; skin; and urinary system. Orthologous to human HOXC4 (homeobox C4). [provided by Alliance of Genome Resources, Nov 2024]',5,'Mus musculus'),(751557,'genomic','15 58.08 cM','microRNA 615','microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]',1,'Mus musculus');
/*!40000 ALTER TABLE `NCBI_table` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `String_table`
--

LOCK TABLES `String_table` WRITE;
/*!40000 ALTER TABLE `String_table` DISABLE KEYS */;
INSERT INTO `String_table` VALUES (16402,'Itgb1','0.999','Fn1','0.999','Itgb3','0.995'),(15422,'Foxn1','0.813','Krt35','0.691','Foxn2','0.672'),(15426,'Hoxc4','0.746','Hoxc9','0.679','Hoxc6','0.673'),(15425,'Hoxc9','0.988','Hoxc5','0.921','Hoxb7','0.885'),(15424,'Hoxc6','0.921','Hoxc4','0.882','Hoxb6','0.742'),(66151,'H2ab3','0.63','Smarcb1','0.627','Actrt1','0.617'),(67488,'Map1lc3b','0.686','Gabarap','0.651','Ccpg1','0.643'),(71726,'Tdg','0.92','Ung','0.897','Mbd4','0.831'),(15423,'Hoxc5','0.882','Hoxb6','0.878','Hoxc6','0.872'),(751557,'Not found','Not found','Not found','Not found','Not found','Not found');
/*!40000 ALTER TABLE `String_table` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `Summary_table`
--

LOCK TABLES `Summary_table` WRITE;
/*!40000 ALTER TABLE `Summary_table` DISABLE KEYS */;
INSERT INTO `Summary_table` VALUES (15422,'ENSMUSG00000001655','Mus musculus','Hoxc13','homeobox C13','genomic',15,'15',102829538,102837249,1,'protein_coding',2,'Enables DNA-binding transcription activator activity, RNA polymerase II-specific and chromatin binding activity. Acts upstream of or within several processes, including hair follicle development; nail development; and tongue morphogenesis. Located in nucleus. Is expressed in several structures, including integumental system; limb; spinal cord; tail; and tongue. Human ortholog(s) of this gene implicated in ectodermal dysplasia 9. Orthologous to human HOXC13 (homeobox C13). [provided by Alliance of Genome Resources, Nov 2024]','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anatomical structure morphogenesis [GO:0009653]; anterior/posterior pattern specification [GO:0009952]; hair follicle development [GO:0001942]; nail development [GO:0035878]; positive regulation of transcription by RNA polymerase II [GO:0045944]; regulation of transcription by RNA polymerase II [GO:0006357]; tongue morphogenesis [GO:0043587]','nucleus [GO:0005634]','chromatin binding [GO:0003682]; DNA-binding transcription activator activity, RNA polymerase II-specific [GO:0001228]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; DNA-binding transcription factor binding [GO:0140297]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); DNA binding (1); Region (1)','Foxn1','0.813','Krt35','0.691','Foxn2','0.672'),(15423,'ENSMUSG00000075394','Mus musculus','Hoxc4','homeobox C4','genomic',15,'15 58.08 cM',102927366,102945278,1,'protein_coding',5,'Enables HMG box domain binding activity. Acts upstream of or within anterior/posterior pattern specification; cartilage development; and embryonic organ morphogenesis. Predicted to be active in nucleoplasm. Is expressed in several structures, including alimentary system; embryo mesenchyme; nervous system; skin; and urinary system. Orthologous to human HOXC4 (homeobox C4). [provided by Alliance of Genome Resources, Nov 2024]','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; cartilage development [GO:0051216]; embryonic organ morphogenesis [GO:0048562]; embryonic skeletal system morphogenesis [GO:0048704]; positive regulation of transcription by RNA polymerase II [GO:0045944]; skeletal system development [GO:0001501]','nucleoplasm [GO:0005654]','DNA-binding transcription activator activity, RNA polymerase II-specific [GO:0001228]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; HMG box domain binding [GO:0071837]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); Compositional bias (3); DNA binding (1); Motif (1); Region (2); Sequence conflict (2)','Hoxc5','0.882','Hoxb6','0.878','Hoxc6','0.872'),(15424,'ENSMUSG00000022485','Mus musculus','Hoxc5','homeobox C5','genomic',15,'15 58.07 cM',102875878,102925861,1,'protein_coding',2,'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Acts upstream of or within anterior/posterior pattern specification and embryonic skeletal system development. Predicted to be located in cell junction and nucleoplasm. Predicted to be active in nucleus. Is expressed in several structures, including alimentary system; central nervous system; embryo mesenchyme; genitourinary system; and respiratory system. Orthologous to human HOXC5 (homeobox C5). [provided by Alliance of Genome Resources, Nov 2024]','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; embryonic skeletal system development [GO:0048706]; regulation of transcription by RNA polymerase II [GO:0006357]','cell junction [GO:0030054]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); DNA binding (1); Motif (1); Region (1); Sequence conflict (2)','Hoxc6','0.921','Hoxc4','0.882','Hoxb6','0.742'),(15425,'ENSMUSG00000001661','Mus musculus','Hoxc6','homeobox C6','genomic',15,'15 58.06 cM',102906689,102920313,1,'protein_coding',4,'Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Acts upstream of or within anterior/posterior pattern specification and embryonic skeletal system development. Predicted to be located in cytosol and nucleoplasm. Predicted to be active in nucleus. Is expressed in several structures, including alimentary system; embryo mesenchyme; genitourinary system; musculoskeletal system; and nervous system. Orthologous to human HOXC6 (homeobox C6). [provided by Alliance of Genome Resources, Nov 2024]','Alternative splicing;Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; embryonic skeletal system development [GO:0048706]; regulation of transcription by RNA polymerase II [GO:0006357]','cytosol [GO:0005829]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Alternative sequence (1); Chain (1); Compositional bias (1); DNA binding (1); Motif (1); Region (1); Sequence conflict (4)','Hoxc9','0.988','Hoxc5','0.921','Hoxb7','0.885'),(15426,'ENSMUSG00000001657','Mus musculus','Hoxc8','homeobox C8','genomic',15,'15 58.05 cM',102899039,102902543,1,'protein_coding',3,'Enables DNA binding activity. Acts upstream of or within several processes, including anterior/posterior pattern specification; negative regulation of transcription by RNA polymerase II; and skeletal system morphogenesis. Located in nucleus. Is expressed in several structures, including alimentary system; embryo mesenchyme; genitourinary system; integumental system; and nervous system. Orthologous to human HOXC8 (homeobox C8). [provided by Alliance of Genome Resources, Nov 2024]','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; negative regulation of transcription by RNA polymerase II [GO:0000122]; neuron differentiation [GO:0030182]; regulation of transcription by RNA polymerase II [GO:0006357]; skeletal system morphogenesis [GO:0048705]','microtubule cytoskeleton [GO:0015630]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA binding [GO:0003677]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II transcription regulatory region sequence-specific DNA binding [GO:0000977]','Chain (1); Compositional bias (3); DNA binding (1); Motif (1); Region (2); Sequence conflict (2)','Hoxc4','0.746','Hoxc9','0.679','Hoxc6','0.673'),(16402,'ENSMUSG00000000555','Mus musculus','Itga5','integrin alpha 5 (fibronectin receptor alpha)','genomic',15,'15 58.9 cM',103252713,103275190,-1,'protein_coding',31,'The product of this gene belongs to the integrin alpha chain family. Integrins are heterodimeric integral membrane proteins composed of an alpha chain and a beta chain. This gene encodes the integrin alpha 5 chain, which is proteolytically processed to generate light and heavy chains that join with beta 1 to form a fibronectin receptor. In addition to adhesion, integrins are known to participate in cell-surface mediated signaling. Integrin alpha 5 and integrin alpha V chains are produced by distinct genes. Homozygous knockout mice for this gene exhibit embryonic lethality. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2015]','Calcium;Cell adhesion;Cell junction;Cell membrane;Cleavage on pair of basic residues;Disulfide bond;Glycoprotein;Integrin;Membrane;Metal-binding;Phosphoprotein;Receptor;Reference proteome;Repeat;Signal;Transmembrane;Transmembrane helix','angiogenesis [GO:0001525]; CD40 signaling pathway [GO:0023035]; cell adhesion [GO:0007155]; cell adhesion mediated by integrin [GO:0033627]; cell-cell adhesion [GO:0098609]; cell-cell adhesion mediated by integrin [GO:0033631]; cell-matrix adhesion [GO:0007160]; cell-substrate junction assembly [GO:0007044]; endodermal cell differentiation [GO:0035987]; female pregnancy [GO:0007565]; heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules [GO:0007157]; heterotypic cell-cell adhesion [GO:0034113]; integrin-mediated signaling pathway [GO:0007229]; leukocyte cell-cell adhesion [GO:0007159]; memory [GO:0007613]; negative regulation of anoikis [GO:2000811]; positive regulation of cell migration [GO:0030335]; positive regulation of cell-substrate adhesion [GO:0010811]; positive regulation of peptidyl-tyrosine phosphorylation [GO:0050731]; positive regulation of sprouting angiogenesis [GO:1903672]; regulation of angiogenesis [GO:0045765]; response to muscle activity [GO:0014850]; wound healing, spreading of epidermal cells [GO:0035313]','cell surface [GO:0009986]; cell-cell junction [GO:0005911]; cytoplasmic vesicle [GO:0031410]; endoplasmic reticulum [GO:0005783]; external side of plasma membrane [GO:0009897]; focal adhesion [GO:0005925]; glutamatergic synapse [GO:0098978]; Golgi apparatus [GO:0005794]; integrin alpha5-beta1 complex [GO:0034674]; integrin complex [GO:0008305]; plasma membrane [GO:0005886]; postsynaptic membrane [GO:0045211]; ruffle membrane [GO:0032587]; synapse [GO:0045202]','calcium ion binding [GO:0005509]; cell adhesion molecule binding [GO:0050839]; epidermal growth factor receptor binding [GO:0005154]; integrin binding [GO:0005178]; signaling receptor activity [GO:0038023]','Binding site (22); Chain (3); Disulfide bond (10); Glycosylation (14); Modified residue (1); Motif (1); Region (2); Repeat (7); Sequence conflict (2); Signal (1); Topological domain (2); Transmembrane (1)','Itgb1','0.999','Fn1','0.999','Itgb3','0.995'),(66151,'ENSMUSG00000023048','Mus musculus','Prr13','proline rich 13','genomic',15,'15 57.6 cM',102367463,102371241,1,'protein_coding',5,'Predicted to be located in cytosol. Predicted to be active in nucleoplasm. Orthologous to human PRR13 (proline rich 13). [provided by Alliance of Genome Resources, Nov 2024]','Nucleus;Reference proteome;Transcription;Transcription regulation','Data not available','nucleoplasm [GO:0005654]','Data not available','Chain (1); Compositional bias (1); Region (2); Sequence conflict (1)','H2ab3','0.63','Smarcb1','0.627','Actrt1','0.617'),(67488,'ENSMUSG00000023055','Mus musculus','Calcoco1','calcium binding and coiled coil domain 1','genomic',15,'15 57.81 cM',102615212,102630613,-1,'protein_coding',16,'Enables chromatin binding activity and transcription coactivator activity. Involved in nuclear receptor-mediated steroid hormone signaling pathway and positive regulation of transcription by RNA polymerase II. Located in chromatin. Is expressed in several structures, including cerebral cortex; choroid plexus; and thymus primordium. Orthologous to human CALCOCO1 (calcium binding and coiled-coil domain 1). [provided by Alliance of Genome Resources, Nov 2024]','Activator;Coiled coil;Cytoplasm;Metal-binding;Nucleus;Phosphoprotein;Reference proteome;Transcription;Transcription regulation;Wnt signaling pathway;Zinc;Zinc-finger','nuclear receptor-mediated steroid hormone signaling pathway [GO:0030518]; positive regulation of gene expression [GO:0010628]; positive regulation of transcription by RNA polymerase II [GO:0045944]; signal transduction [GO:0007165]; Wnt signaling pathway [GO:0016055]','chromatin [GO:0000785]; cytosol [GO:0005829]; nucleus [GO:0005634]','beta-catenin binding [GO:0008013]; chromatin binding [GO:0003682]; metal ion binding [GO:0046872]; nuclear receptor coactivator activity [GO:0030374]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]; transcription coactivator activity [GO:0003713]','Binding site (4); Chain (1); Coiled coil (3); Compositional bias (1); Modified residue (1); Mutagenesis (3); Region (5); Sequence conflict (1); Zinc finger (1)','Map1lc3b','0.686','Gabarap','0.651','Ccpg1','0.643'),(71726,'ENSMUSG00000036061','Mus musculus','Smug1','single-strand selective monofunctional uracil DNA glycosylase','genomic',15,'15 58.2 cM',103061717,103075519,-1,'protein_coding',12,'Predicted to enable DNA N-glycosylase activity; identical protein binding activity; and single-stranded DNA binding activity. Predicted to be involved in base-excision repair. Predicted to be located in cytosol; fibrillar center; and nucleoplasm. Is expressed in several structures, including lower urogenital tract; palatal shelf; reproductive system; retina layer; and urethra epithelium. Orthologous to human SMUG1 (single-strand-selective monofunctional uracil-DNA glycosylase 1). [provided by Alliance of Genome Resources, Nov 2024]','DNA damage;DNA repair;DNA-binding;Hydrolase;Nucleus;Reference proteome','base-excision repair [GO:0006284]; DNA repair [GO:0006281]','cytosol [GO:0005829]; fibrillar center [GO:0001650]; nucleolus [GO:0005730]; nucleoplasm [GO:0005654]','DNA N-glycosylase activity [GO:0019104]; oxidized base lesion DNA N-glycosylase activity [GO:0000702]; oxidized pyrimidine nucleobase lesion DNA N-glycosylase activity [GO:0000703]; single-strand selective uracil DNA N-glycosylase activity [GO:0017065]; single-stranded DNA binding [GO:0003697]; uracil DNA N-glycosylase activity [GO:0004844]','Binding site (4); Chain (1); Region (1); Sequence conflict (1)','Tdg','0.92','Ung','0.897','Mbd4','0.831'),(751557,'ENSMUSG00000076010','Mus musculus','Mir615','microRNA 615','genomic',15,'15 58.08 cM',102923342,102923433,1,'miRNA',1,'microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]','Not found','Not found','Not found','Not found','Not found','Not found','Not found','Not found','Not found','Not found','Not found');
/*!40000 ALTER TABLE `Summary_table` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Dumping data for table `Uniprot_table`
--

LOCK TABLES `Uniprot_table` WRITE;
/*!40000 ALTER TABLE `Uniprot_table` DISABLE KEYS */;
INSERT INTO `Uniprot_table` VALUES ('Itga5','Calcium;Cell adhesion;Cell junction;Cell membrane;Cleavage on pair of basic residues;Disulfide bond;Glycoprotein;Integrin;Membrane;Metal-binding;Phosphoprotein;Receptor;Reference proteome;Repeat;Signal;Transmembrane;Transmembrane helix','angiogenesis [GO:0001525]; CD40 signaling pathway [GO:0023035]; cell adhesion [GO:0007155]; cell adhesion mediated by integrin [GO:0033627]; cell-cell adhesion [GO:0098609]; cell-cell adhesion mediated by integrin [GO:0033631]; cell-matrix adhesion [GO:0007160]; cell-substrate junction assembly [GO:0007044]; endodermal cell differentiation [GO:0035987]; female pregnancy [GO:0007565]; heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules [GO:0007157]; heterotypic cell-cell adhesion [GO:0034113]; integrin-mediated signaling pathway [GO:0007229]; leukocyte cell-cell adhesion [GO:0007159]; memory [GO:0007613]; negative regulation of anoikis [GO:2000811]; positive regulation of cell migration [GO:0030335]; positive regulation of cell-substrate adhesion [GO:0010811]; positive regulation of peptidyl-tyrosine phosphorylation [GO:0050731]; positive regulation of sprouting angiogenesis [GO:1903672]; regulation of angiogenesis [GO:0045765]; response to muscle activity [GO:0014850]; wound healing, spreading of epidermal cells [GO:0035313]','cell surface [GO:0009986]; cell-cell junction [GO:0005911]; cytoplasmic vesicle [GO:0031410]; endoplasmic reticulum [GO:0005783]; external side of plasma membrane [GO:0009897]; focal adhesion [GO:0005925]; glutamatergic synapse [GO:0098978]; Golgi apparatus [GO:0005794]; integrin alpha5-beta1 complex [GO:0034674]; integrin complex [GO:0008305]; plasma membrane [GO:0005886]; postsynaptic membrane [GO:0045211]; ruffle membrane [GO:0032587]; synapse [GO:0045202]','calcium ion binding [GO:0005509]; cell adhesion molecule binding [GO:0050839]; epidermal growth factor receptor binding [GO:0005154]; integrin binding [GO:0005178]; signaling receptor activity [GO:0038023]','Binding site (22); Chain (3); Disulfide bond (10); Glycosylation (14); Modified residue (1); Motif (1); Region (2); Repeat (7); Sequence conflict (2); Signal (1); Topological domain (2); Transmembrane (1)'),('Hoxc13','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anatomical structure morphogenesis [GO:0009653]; anterior/posterior pattern specification [GO:0009952]; hair follicle development [GO:0001942]; nail development [GO:0035878]; positive regulation of transcription by RNA polymerase II [GO:0045944]; regulation of transcription by RNA polymerase II [GO:0006357]; tongue morphogenesis [GO:0043587]','nucleus [GO:0005634]','chromatin binding [GO:0003682]; DNA-binding transcription activator activity, RNA polymerase II-specific [GO:0001228]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; DNA-binding transcription factor binding [GO:0140297]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); DNA binding (1); Region (1)'),('Hoxc8','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; negative regulation of transcription by RNA polymerase II [GO:0000122]; neuron differentiation [GO:0030182]; regulation of transcription by RNA polymerase II [GO:0006357]; skeletal system morphogenesis [GO:0048705]','microtubule cytoskeleton [GO:0015630]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA binding [GO:0003677]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II transcription regulatory region sequence-specific DNA binding [GO:0000977]','Chain (1); Compositional bias (3); DNA binding (1); Motif (1); Region (2); Sequence conflict (2)'),('Hoxc6','Alternative splicing;Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; embryonic skeletal system development [GO:0048706]; regulation of transcription by RNA polymerase II [GO:0006357]','cytosol [GO:0005829]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Alternative sequence (1); Chain (1); Compositional bias (1); DNA binding (1); Motif (1); Region (1); Sequence conflict (4)'),('Hoxc5','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; embryonic skeletal system development [GO:0048706]; regulation of transcription by RNA polymerase II [GO:0006357]','cell junction [GO:0030054]; nucleoplasm [GO:0005654]; nucleus [GO:0005634]','DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); DNA binding (1); Motif (1); Region (1); Sequence conflict (2)'),('Prr13','Nucleus;Reference proteome;Transcription;Transcription regulation','Data not available','nucleoplasm [GO:0005654]','Data not available','Chain (1); Compositional bias (1); Region (2); Sequence conflict (1)'),('Calcoco1','Activator;Coiled coil;Cytoplasm;Metal-binding;Nucleus;Phosphoprotein;Reference proteome;Transcription;Transcription regulation;Wnt signaling pathway;Zinc;Zinc-finger','nuclear receptor-mediated steroid hormone signaling pathway [GO:0030518]; positive regulation of gene expression [GO:0010628]; positive regulation of transcription by RNA polymerase II [GO:0045944]; signal transduction [GO:0007165]; Wnt signaling pathway [GO:0016055]','chromatin [GO:0000785]; cytosol [GO:0005829]; nucleus [GO:0005634]','beta-catenin binding [GO:0008013]; chromatin binding [GO:0003682]; metal ion binding [GO:0046872]; nuclear receptor coactivator activity [GO:0030374]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]; transcription coactivator activity [GO:0003713]','Binding site (4); Chain (1); Coiled coil (3); Compositional bias (1); Modified residue (1); Mutagenesis (3); Region (5); Sequence conflict (1); Zinc finger (1)'),('Smug1','DNA damage;DNA repair;DNA-binding;Hydrolase;Nucleus;Reference proteome','base-excision repair [GO:0006284]; DNA repair [GO:0006281]','cytosol [GO:0005829]; fibrillar center [GO:0001650]; nucleolus [GO:0005730]; nucleoplasm [GO:0005654]','DNA N-glycosylase activity [GO:0019104]; oxidized base lesion DNA N-glycosylase activity [GO:0000702]; oxidized pyrimidine nucleobase lesion DNA N-glycosylase activity [GO:0000703]; single-strand selective uracil DNA N-glycosylase activity [GO:0017065]; single-stranded DNA binding [GO:0003697]; uracil DNA N-glycosylase activity [GO:0004844]','Binding site (4); Chain (1); Region (1); Sequence conflict (1)'),('Hoxc4','Developmental protein;DNA-binding;Homeobox;Nucleus;Reference proteome;Transcription;Transcription regulation','anterior/posterior pattern specification [GO:0009952]; cartilage development [GO:0051216]; embryonic organ morphogenesis [GO:0048562]; embryonic skeletal system morphogenesis [GO:0048704]; positive regulation of transcription by RNA polymerase II [GO:0045944]; skeletal system development [GO:0001501]','nucleoplasm [GO:0005654]','DNA-binding transcription activator activity, RNA polymerase II-specific [GO:0001228]; DNA-binding transcription factor activity, RNA polymerase II-specific [GO:0000981]; HMG box domain binding [GO:0071837]; RNA polymerase II cis-regulatory region sequence-specific DNA binding [GO:0000978]','Chain (1); Compositional bias (3); DNA binding (1); Motif (1); Region (2); Sequence conflict (2)'),('Mir615','Not found','Not found','Not found','Not found','Not found');
/*!40000 ALTER TABLE `Uniprot_table` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2024-11-12  4:11:09
